engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -1117,7 +1883,7 @@ public void populateDAG(LogicalPlan dag)
             nd = dag.addOperator(nodeConfEntry.getKey(), nodeClass);
           }
           setOperatorProperties(nd, nodeConf.getProperties());
-        } catch (Exception e) {
+        } catch (IOException e) {
           throw new IllegalArgumentException("Error setting operator properties " + e.getMessage(), e);
```

Comment:
Why do we throw an IllegalArgumentException instead of rethrowing an IO exception here?